tests/linker/ios/link sdk/LinkSdkRegressionTest.cs

```diff
@@ -172,7 +174,7 @@ public void Bug928_MonoModule_LinkedOut ()
 			Assert.That (m.GetType ().Name, Is.EqualTo ("RuntimeModule"), "RuntimeModule");
 		}
 
-#if !__TVOS__ && !__WATCHOS__
+#if HAS_ADDRESSBOOKUI
```

Comment:
Does this needs a dual condition?

```suggestion
#if HAS_ADDRESSBOOK && HAS_ADDRESSBOOKUI
```

Reply:
I took a shortcut as `AddressBookUI` implies `AddressBook` but there's no harm in being explicit